v0/destinations/blueshift/data/blueshiftIdentifyConfig.json

```
@@ -0,0 +1,47 @@
+[
+  {
+    "destKey": "email",
+    "sourceKeys": "email",
+    "sourceFromGenericMap": true,
+    "required": true
+  },
+  {
+    "destKey": "customer_id",
+    "sourceKeys": "userIdOnly",
+    "sourceFromGenericMap": true,
+    "required": true
+  },
+  {
+    "destKey": "event",
+    "sourceKeys": "event",
+    "required": false
+  },
+  {
+    "destKey": "phone_number",
+    "sourceKeys": "phone",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "firstname",
+    "sourceKeys": "firstName",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "lastname",
+    "sourceKeys": "lastName",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "gender",
+    "sourceKeys": ["traits.gender", "context.traits.gender"],
+    "required": false
+  },
+  {
+    "destKey": "custom_attributes",
+    "sourceKeys": ["context.traits", "traits"],
+    "required": false
```

```suggestion
{
    "destKey": "custom_attributes",
    "sourceKeys": ["context.traits", "traits"],
    "required": false,
    "metadata": {
      "excludes": [
       "email",
       "userId",
       "phone",
       "firstName",
       "firstname",
       "first_name",
       "lastName",
       "lastname",
       "last_name",
        "gender"
      ]
    }
   }
```

We should exclude the fields from custom_attributes that are already mapped